Zapper-fi/studio
# feat(inverse-firm): Add cvxFxs rewards

## Description

Add the CVX+FXS rewards for the cvxFXS FiRM market

## Checklist


## How to test?
Via the inverse website:
For the user `0xAAc0aa431c237C2C0B5f041c8e59B3f1a43aC78F` rewards should match with what's displayed here (need wallet connected): https://www.inverse.finance/firm/cvxFXS?viewAddress=0xAAc0aa431c237C2C0B5f041c8e59B3f1a43aC78F

Via etherscan:
For the user `0xAAc0aa431c237C2C0B5f041c8e59B3f1a43aC78F` rewards should match with
the `claimableRewards` read function (the escrow address to use for the call for that user: `0xE873174F2988c4C1eEE3D820b30F1f8cD24cA220`) so `claimableRewards(0xE873174F2988c4C1eEE3D820b30F1f8cD24cA220)`
https://etherscan.io/address/0x49b4d1dF40442f0C31b1BbAEA3EDE7c38e37E31a#readContract
